sample-apps/donet-sample-app/Controllers/MetricEmitter.cs

```diff
@@ -79,26 +64,40 @@ public MetricEmitter()
             }
             
 
-            meter.CreateObservableCounter(totalApiRequestSent,() => apiRequestSent, 
+            meter.CreateObservableCounter(totalApiRequestSent,() => { 
+                    return new Measurement<long>(apiRequestSent, new KeyValuePair<string, object>[] {
+                        new KeyValuePair<string, object>("signal", "metric"), 
+                        new KeyValuePair<string, object>("language", "dotnet"), 
+                        new KeyValuePair<string, object>("metricType", "request")});
```

Comment:
Nit: We reuse two sets of key-value pair triples throughout this class. Is there anyway we can DRY this up?

Reply:
I have made two array variables containing the attributes and reuse those instead.